eng/targets/FunctionalTestAsset.targets

```diff
@@ -1,9 +0,0 @@
-<Project>
```

Comment:
Are we sure we want to delete these?  I believe the idea for these is to generalize what the IIS functionals are doing, so if we were ever going to move the rest of the hosting functionals etc onto helix, we would need something like these.